frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -86,7 +86,10 @@ export const getViewerState = async () => {
 	}
 
 	if (viewpointV4.highlighted_group) {
-		state.colored = (state.colored || []).concat(convertToV5GroupOverride(viewpointV4.highlighted_group, ViewpointGroupOverrideType.COLORED));
+		state.colored = (state.colored || [])
+			.concat(convertToV5GroupOverride(viewpointV4.highlighted_group, ViewpointGroupOverrideType.COLORED))
+			// eslint-disable-next-line no-underscore-dangle
+			.filter(({ group }) => !!(group as Group).objects.filter((obj) => obj._ids.length).length);
```

Comment:
Why did you change this?

Reply:
To fix this bug
![image](https://github.com/3drepo/3drepo.io/assets/43547703/b2dc87c4-a237-4b58-83b9-d4b0fec7c415)